video_emotion/facial_recognition/facial_recognition.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import cv2
+import os
+
+dirname = os.path.dirname(__file__)
+CONFIDENCE_MINIMUM = 0.7
+
+opencv_prototxt = os.path.join(
+        dirname, "deploy.prototxt.txt")
+
+opencv_model = os.path.join(
+        dirname, "res10_300x300_ssd_iter_140000_fp16.caffemodel")
+
+# Load model from disk
+net = cv2.dnn.readNetFromCaffe(opencv_prototxt, opencv_model)
+
+def analyze_video(video_path, time_range=None):
+    if time_range is not None:
+        fro, to = time_range
+    cap = cv2.VideoCapture(video_path)
+    dict_of_faces = {}
+    while(cap.isOpened()):
+        # Get frame from video
+        ret, frame = cap.read()
+
+        # Ignore frames that are not within the given time_range
+        if time_range is not None:
+            timestamp = cap.get(cv2.CAP_PROP_POS_MSEC)
+            if timestamp < fro:
+                continue
+            elif timestamp > to:
+                break
+
+        # Check if we received a frame
+        if ret is False:
+            break
+
+        # Get faces from frame
+        faces = analyze_frame(frame)
+
+        # Add found frames to our dictionary
+        if len(faces) > 0:
+            frame_number = int(cap.get(cv2.CAP_PROP_POS_FRAMES))
+            dict_of_faces[str(frame_number)] = faces
+    # Release resources used to open video
+    cap.release()
+
+    return dict_of_faces
+
+
+def analyze_frame(frame):
+    # Get width and height of frame
+    (h, w) = frame.shape[:2]
+
+    # Resize frame and load it as blob
+    blob = cv2.dnn.blobFromImage(cv2.resize(frame, (300, 300)), 1.0, (300, 300), (104.0, 177.0, 123.0))
```

Comment:
Would be nice with an explanation of the numbers used in ```cv2.dnn.blobFromImage```.

If it is values, that we might change, add them as constants at the top of the file.